src/CountUp.tsx

```diff
@@ -119,49 +125,54 @@ class CountUp extends Component {
     // end value.
     if (end !== prevProps.end) {
       if (!preserveValue) {
-        this.instance.reset();
+        this.instance!.reset();
       }
-      this.instance.update(end);
+      this.instance!.update(end);
     }
   }
 
   componentWillUnmount() {
     if (this.timeoutId) {
       clearTimeout(this.timeoutId);
     }
-    this.instance.reset();
+    this.instance?.reset();
   }
 
   createInstance = () => {
     if (typeof this.props.children === 'function') {
       // Warn when user didn't use containerRef at all
       warning(
-        this.containerRef.current &&
-          (this.containerRef.current instanceof HTMLElement ||
-            this.containerRef.current instanceof SVGTextElement ||
-            this.containerRef.current instanceof SVGTSpanElement),
+        this.containerRef.current instanceof HTMLElement ||
+          !!(
+            this.containerRef.current &&
+            AVAILABLE_SVG_TAGS.includes(
```

Comment:
Let's put the list of svg tags directly here as long as it's not used elsewhere.